python/ray/autoscaler/autoscaler.py

```diff
@@ -663,10 +674,10 @@ def recover_if_needed(self, node_id, now):
             auth_config=self.config["auth"],
             cluster_name=self.config["cluster_name"],
             file_mounts={},
-            initialization_commands=[],
-            setup_commands=[],
-            ray_start_commands=with_head_node_ip(
-                self.config["worker_start_ray_commands"]),
+            node_setup_commands=[],
+            node_restart_commands=[],
+            ray_restart_commands=with_head_node_ip(
+                get_commands(self.config, "ray_restart_commands", head=False)),
```

Comment:
I'm unsure about this. It would make renaming things easier (can use IDEs to automatically find usage), but I assume we don't want to do that often as it will break users configs? There are also a bunch of uses it won't capture (e.g. in example YAML files), so a find-and-replace would still be necessary.

If you still feel this is worth doing I'm OK making the change, though.

Reply:
Better to have it as consolidated as possible IMO. At least this would mean only looking for it/changing it one place in the code itself (and next to the schema instead of scattered throughout the codebase), changing YAML files is unavoidable without some sort of templating magic.

Reply:
Done